frontend/src/components/BottomNavigationBar.tsx

```diff
@@ -0,0 +1,26 @@
+import {BottomNavigation, BottomNavigationAction, Box, Paper} from "@mui/material";
+import {Link, useLocation} from "react-router-dom";
+import {SyntheticEvent, useState} from "react";
+import {MapSharp, Search} from "@mui/icons-material";
+
+export default function BottomNavigationBar(){
+    const pathname = useLocation().pathname
+    const [value, setValue] = useState(pathname)
+
+
+    function onChange(event: SyntheticEvent<Element, Event>, newValue: any) {
```

Comment:
`newValue: any` Sooo, what is the newValue's type :wink: ?
Also: Let's keep up the `handle...` convention!